help/de/Help_ch_Account-Actions.xml

```diff
@@ -829,61 +839,103 @@
                   </para>
-                </note>
 
-                <para>Beachten Sie allerdings, dass Sie bei <quote>Yahoo Europe</quote> zusätzlich das
-                  Börsenplatzkürzel des Wertpapiers anfügen sollten, z.B. <quote>PA</quote> für
-                  Paris, <quote>BE</quote> für Berlin usw. Beispiel: 12150.PA (ein Peugeot-Papier
-                  am Pariser Markt). <xref linkend="gnc-tbl-fq-yahoo-exchange-codes"></xref>
-                  enthält eine Liste von Suffixen für verschiedene Märkte auf der ganzen Welt.
-                </para>
+                  <variablelist spacing="compact">
+                    <varlistentry>
+                      <term><guilabel>Einzel</guilabel></term>
+
+                      <listitem>
+                        <para>Diese Auswahl ruft die Daten nur aus der angegebenen Quelle ab.
+                        </para>
+                      </listitem>
+                    </varlistentry>
+
+                    <varlistentry>
+                      <term><guilabel>Mehrere Kursquellen</guilabel></term>
+
+                      <listitem>
+                        <para>Kursquellen wie <quote>Europa</quote> sollten verwendet werden, wenn eine Ausfallsicherung
+                          gegenüber mehreren Datenquellen erwünscht ist.
+                        </para>
+
+                        <caution>
+                          <para>Die Option <guilabel>Mehrere Kursquellen</guilabel> ist nur sinnvoll, wenn alle ausgewählten
+                            Quellen das gleiche Symbol für Ihre Wertpapier verwenden.
+                          </para>
+                        </caution>
+                      </listitem>
+                    </varlistentry>
+
+                    <varlistentry>
+                      <term><guilabel>unbekannt</guilabel></term>
+<!-- FIXME source: What function does this option have? -->
+                      <listitem>
+                        <para></para>
+                      </listitem>
+                    </varlistentry>
+                  </variablelist>
+
+                  <para>Nachdem Sie die Art der Quelle für Kursangebote ausgewählt haben, wählen Sie eine Datenquelle aus
+                    dem Pulldown-Menü. Details zu den derzeit unterstützten Datenquellen finden
+                    Sie unter <xref linkend="fq-sources"></xref>.
+                  </para>
 
-                <para>Pseudo-Symbole für TIAA-CREF-Fonds sind in der Tabelle <xref linkend="PsSymbs-TIAA-CREF"></xref>
-                  aufgeführt.
-                </para>
-              </listitem>
+                  <note>
+                    <para>Die Kursdatenquelle <quote>Yahoo</quote> liefert dabei auch die Kurse für die übrigen genannten
+                      Unternehmen (z.B. Fidelity Investments, T.Rowe Price und The Vanguard Group)
+                      und gibt den Kurs normalerweise zeitverzögert zur Original-Datenquelle an.
```

Comment:
3 verschiedene Sachen:
Einige Quellen wie Vanguard sind lediglich Aliasse für Alphavantage (früher yahoo. In den Perl Modulen nachsehen, wenn es im Anhang nicht steht).

Einige Papiere lassen sich über verschiedene Quellen abrufen. Von Yahoo kennen wir die Liste der Verzögerungen, die von AV dürften ähnlich sein. Habe ich aber auch den Emittenten des Papiers als Quelle, sollte es da keine Verzögerung geben.

Sowohl AV als auch Yahoo, gehen vom amerikanischen Markt aus, wenn kein Suffix angegeben ist. Siehe im Wiki den Versuch, den Kurs des indischen "Mahindra & Mahindra"-Konzerns abzurufen.

Reply:
@fellen: hierfür darfst du mir auch noch gerne einen Textvorschlag formulieren. ;-)